README.md

```diff
@@ -174,6 +148,24 @@ It accepts command-line arguments, listed by:
 scrcpy --help
 ```
 
+
+## Custom paths
+
+(Linux): For Android Studio (SDK) `adb` to be `adb` command, do `sudo ln --symbolic ### /usr/local/bin/adb`.  
```

Comment:
#3063 .

Reply:
I'm not sure this is the right place to explain this (it is independent of scrcpy).

Either you use the default `adb` (wherever it comes from in your PATH), either you can overwrite it using the `ADB` environment variable.